### PR TITLE
Day label created

### PR DESCRIPTION
**Pull request by adhamarif**
Here are few updates from me.

1) The `input.py` from #13 renamed to `user_input.py` because `input` is the reserved keyword in python. I renamed the file because I imported some function from `input.py` in #13. It is not possible to import the module if the filename contain reserved keyword. 

2) First version of `day_label` input is created. Most of the code block is inspired from @till-teb code in #13. It only has basic function for now, which ask the label for today. Further features will be added later.

Thank you!

### CODE REVIEW

**Comment by adhamarif** [comment]
Issue #15 resolved here

File: expenses_management_tool/day_label.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        
+        """
+        df = split_DATE(df)  # day, month, year as new columns
+        print("--saved--")
+        to_drop = ["DATE"]  # only columns: day, month, year column are nessesary
+        df = df.drop(to_drop, axis=1)
+        
+        folder = "datasets"
+        folder_PATH = os.path.join(root, folder) 
+        if not os.path.exists(folder_PATH):
```
Comment (till-teb) [inline expenses_management_tool/day_label.py:63]: uhh, I really like your solution for that problem :-)

**Review by till-teb — APPROVED** [review]
Adham, great work. The function looks great 👏

**Comment by till-teb** [comment]
@adhamarif :

I can imagine that the users from the `expenses_management_tool` do not typ in their expenses every day. That is a Problem, because our function is so great, that it generates the current date automatically. Therefore:

What do you think can we do to solve that problem?

**Comment by adhamarif** [comment]
yep that is my concern too. My idea is to add extra feature which allow user to type the label for another date. With this, user can type the label to the day they forgot to give label. This also allows user to change label for certain date. 

```python
def other_day_input(date)
"""
This function give label to the day for the given date.
"""
```

So lets say if I forgot to give label for yesterday, I can enter the date for yesterday and give label.

**Comment by adhamarif** [comment]
I also have an idea to improve the function. For now, user can add so many label for a day which we didnt want that.

One day should have only one label.

So another improvement is, to make a check function, so that a date only have one label.

**Comment by till-teb** [comment]
Yeah, great ideas.👏 I m thankful to read your answers. Let´s build that function, I mean at least its needed for two functions right?

What do you think about a small user input to ask the user: what function do you want to use?

- set the date for today          |--> than we generate the date automatically 
or
- set the date for yesterday |--> than we can use the new function you perfectly described 

What do you think @adhamarif ?

**Pull request merged**